app/components/markdown/latex_code_block/latex_code_block.js

```diff
@@ -0,0 +1,209 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {PropTypes} from 'prop-types';
+import React from 'react';
+import {Keyboard, View, Text, StyleSheet, Platform} from 'react-native';
+import MathView from 'react-native-math-view';
+
+import {goToScreen} from '@actions/navigation';
+import FormattedText from '@components/formatted_text';
+import TouchableWithFeedback from '@components/touchable_with_feedback';
+import {splitLatexCodeInLines} from '@utils/latex';
+import {getDisplayNameForLanguage} from '@utils/markdown';
+import {preventDoubleTap} from '@utils/tap';
+import {changeOpacity, makeStyleSheetFromTheme} from '@utils/theme';
+
+import MarkdownCodeBlock from '../markdown_code_block/markdown_code_block';
+
+const MAX_LINES = 2;
+
+export default class LatexCodeBlock extends MarkdownCodeBlock {
+    static propTypes = {
+        theme: PropTypes.object.isRequired,
+        content: PropTypes.string.isRequired,
+        textStyle: PropTypes.oneOfType([PropTypes.object, PropTypes.number, PropTypes.array]),
+    };
+
+    handlePress = preventDoubleTap(() => {
+        const {language, content} = this.props;
+        const {intl} = this.context;
+        const screen = 'Latex';
+        const passProps = {
+            content,
+        };
+
+        const languageDisplayName = getDisplayNameForLanguage(language);
```

Comment:
To follow up on https://github.com/mattermost/mattermost-mobile/pull/5651#discussion_r718550915, could you remove the language prop from here as well?